libcodechecker/server/api/report_server.py

```diff
@@ -712,11 +729,18 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
             if cmp_data:
                 diff_hashes, run_ids = self._cmp_helper(session,
                                                         run_ids,
+                                                        report_filter,
                                                         cmp_data)
                 if not diff_hashes:
                     # There is no difference.
                     return results
 
+                # If we are in diff mode and run tag is set in compare data,
+                # after diff hashes are calculated remove it from the
+                # report filter.
+                if cmp_data.runTag:
+                    report_filter.runTag = None
```

Comment:
Could you describe it in more detail why do we need to remove the runTag from the filter?